### DISCUSSION TITLE
[Esame 17 aprile 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#459] [Esami M-Z]
![17 apr es.2](https://github.com/user-attachments/assets/e1e72d26-cd75-437c-baf2-ed1f816e2fab)

**Comment by LeonardoMarelli** [comment]
![17 ](https://github.com/user-attachments/assets/bda7e258-0d13-405e-95c9-fff34940f845)
una matrice non e inv quando det = 0 *

**Comment by niccolo-fato** [comment]
<img width="1219" height="1208" alt="image" src="https://github.com/user-attachments/assets/a1623a5d-9bd0-4f06-966e-7628296cf77d" />
